lib/cookie_consent_functions.php

```diff
@@ -71,7 +72,8 @@ public static function cookie_consent_backend() {
                 "deny"    => rex_escape($deny_content),
                 "allow"   => rex_escape($allow_content),
                 "link"    => rex_escape($link_content),
-                "href"    => rex_escape($externer_link) . '' . rex_escape($interner_link)
+                "href"    => rex_escape($externer_link) . '' . rex_escape($interner_link),
+				"target"  => rex_escape($link_target_type)
```

Comment:
Sollte wie in [L167](https://github.com/FriendsOfREDAXO/cookie_consent/pull/20/files#diff-037ad815c25266674072ced92eb0c0d1R167)